sentry_streams/sentry_streams/runner.py

```diff
@@ -146,6 +177,24 @@ def signal_handler(sig: int, frame: Any) -> None:
     type=str,
     help="The segment id to run the pipeline for",
 )
+@click.option(
```

Comment:
i think the tricky bit is how we'd get runtime metrics and application metrics tagged with the same set of global tags.

maybe there's an API in the streaming platform to get a set of global tags that the applpication can then add to its own config.

right now we do it the other way around (which I think is wrong): sentry has a mechanism for setting global tags, and imposes it onto arroyo.

Reply:
Oh I see. There is actually a place for global tags to also be defined, when the metrics backend is instantiated. Those tags are injected into every metric call. I will add a `add_global_tag` function so that those tags can be modified dynamically.